pkg/controller/csr_check.go

```diff
@@ -231,47 +249,63 @@ func authorizeCSR(
 		}
 		// The CSR requested an IP name that did not belong to the machine
 		if !foundSan {
-			return fmt.Errorf("IP address '%s' not in machine addresses: %s", san, strings.Join(attemptedAddresses, " "))
+			//TODO: set annotation/emit event here.
+			// return error so we requeue, in case machine network is out of date
+			// for some reason
+			klog.Errorf("%v: IP address '%s' not in machine addresses: %s", req.Name, san, strings.Join(attemptedAddresses, " "))
```

Comment:
Are there security implications of logging things like IP addresses?

Reply:
No.